src/Native/LibTorchSharp/crc32c.c

```diff
@@ -0,0 +1,930 @@
+/* Part of CRC-32C library: https://crc32c.machinezoo.com/ */
```

Comment:
>Part of CRC-32C library: https://crc32c.machinezoo.com/ */ [](http://example.com/codeflow?start=3&length=58)

do we want to add something to [https://github.com/dotnet/TorchSharp/blob/main/THIRD-PARTY-NOTICES.TXT](https://github.com/dotnet/TorchSharp/blob/main/THIRD-PARTY-NOTICES.TXT) about this?

Reply:
Yes, of course. Thanks for catching that.